.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug Report
+about: Something isn't working as expected? Create a report to help us improve.
+labels: "c.Bug"
+---
+
+<!--
+  Before opening a new issue, please search existing issues:  https://github.com/reposense/RepoSense/issues
+-->
+
+**Tell us about your environment**
+
+* **RepoSense Version:**
```

Comment:
Thanks for setting this up @yash-chowdhary 
I think we should also request the bug reporter to provide their web browser and operating system details.